src/main/java/com/techiteasy/demo/controllers/ExceptionController.java

```diff
@@ -0,0 +1,30 @@
+package com.techiteasy.demo.controllers;
+
+import com.techiteasy.demo.exceptions.RecordNotFoundException;
+import com.techiteasy.demo.exceptions.TelevisionNameTooLongException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+//Deze klasse behandelt uitzonderingen (zoals de RecordNotFoundException)
+@Controller // Stap 5: voorzie deze klasse van de juiste annotatie
```

Comment:
De exception controller heeft de annotatie `@ControllerAdvice` nodig. Het is namelijk een helper voor je controllers, geen echte controller (er staan geen endpoints in die een gebruiker direct kan aanspreken)